lib/moon_web/components/page.ex

```diff
@@ -13,7 +13,7 @@ defmodule MoonWeb.Components.Page do
   prop class, :css_class
   prop breadcrumbs, :any
   prop theme_name, :any, default: "moon-design-light"
-  prop direction, :string, values: ["ltr", "rtl"], default: "ltr"
+  prop direction, :any, values: ["ltr", "rtl"], default: "ltr"
```

Comment:
why :any, why not :string here?

Reply:
oh sorry, I was debugging and forgot to revert. I will revert in my next PR